package.json

```diff
@@ -155,7 +155,7 @@
     "react-native-gesture-handler": "^1.10.3",
     "react-native-keychain": "^7.0.0",
     "react-native-level-fs": "^3.0.0",
-    "react-native-locale": "0.0.19",
+    "react-native-localize": "^2.2.1",
```

Comment:
- Replacing `react-native-locale` (old 4y outdated lib) by `react-native-localize` (more modern, **gives us access to the system timezone**).